packages/jest-console/src/BufferedConsole.ts

```diff
@@ -79,7 +79,7 @@ export default class BufferedConsole extends Console {
       if (!(error instanceof AssertionError)) {
         throw error;
       }
-      this._log('assert', error.toString());
+      this._log('assert', error.toString().replace(/:\n\n.*\n/gs, ''));
```

Comment:
this seems very weird to me - could you add a comment above this (and the similar one in `CustomConsole`) why it's neeed?

Reply:
Can I link to the PR?

Here I don't know why nodejs is outputting extraneous source code, but it shouldn't matter.
1. Our assert is wrapped, nodejs can never output user code text.
2. Previously nodejs couldn't even get any code text.
So when nodejs outputs an extraneous and wrong code text we have no problem removing it.

Reply:
>Can I link to the PR?

Sure!